client/src/main/scala/drt/client/components/styles/ToolTipsStyle.scala

```diff
@@ -0,0 +1,18 @@
+package drt.client.components.styles
+
+import drt.client.components.styles.ScalaCssImplicits.CssSettings._
+import scalacss.internal.mutable.StyleSheet
+
+case class ToolTipsStyle(common: CommonStyle = DefaultCommonStyle) extends StyleSheet.Inline {
+
+  import common.theme
+  import dsl._
+
+  val triggerHoverIndicator = style(
+    cursor.pointer,
+    &.hover(opacity(0.75))
+  )
+
+}
+
+object DefaultToolTipsStyle extends ToolTipsStyle
```

Comment:
If this is defaultToolTip should CommonStyle = DefaultCommonStyle pass here rather than default param value ?

Reply:
this is just following the MaterialUI pattern, I'm not sure what the right way to do it is.

Reply:
I thought default parameter value makes is default and creating object just make singleton  .

Reply:
that's correct - I think this probably is a reasonable pattern, it leaves the option to extend it (e.g. if you wanted to make a dark theme, but also makes it easy to implement. I'm still learning how ScalaCSS works though really.